.github/workflows/ci.yml

```diff
@@ -47,7 +47,7 @@ jobs:
             cmd: make -C doc/ html
             type: lint
           - name: pytest
-            cmd: pytest --cov=pymodbus --cov=test --cov-report=term-missing --cov-report=xml -v --full-trace --timeout=20
+            cmd: pytest --cov=pymodbus --cov=test --cov-report=term-missing --cov-report=xml -v --full-trace --timeout=20 --numprocesses auto
```

Comment:
If this works it is pr accident. Multiple tests open a listener on port 5020, if they run in parallel they will fail.